### ISSUE TITLE
What is the optimal configuration to train the model from scratch?

### ISSUE THREAD

**Issue by shubhamwagh** [weitong8591/differentiable_ransac#1]
This work is really interesting! 
I am investigating the use of this package for a monocular VO based problem I am current working on using ground truth poses which is mainly dealing with estimating relative camera poses.

Given that I have extracted all image pair key point correspondences and features, may I know what do you recommend to train differential RANSAC model completely from scratch for Fundamental Matrix estimation? 

Thanks.

**Comment by weitong8591** [comment]
Hi! Thanks for your interest. In terms of the fundamental matrix, I would recommend trying to use the epipolar error as the loss (w2) , and 8PC as the solver, with the initialized weights, and learning rate 1e-5. Also, try it with the 7PC solver. 

You can also refer to our paper, https://arxiv.org/abs/2212.13185

Tong
